crates/bevy_pbr/src/render/view_transformations.wgsl

```diff
@@ -0,0 +1,170 @@
+#define_import_path bevy_pbr::view_transformations
+#import bevy_pbr::mesh_view_bindings as view_bindings
+
+/// World space:
+/// +y is up
+
+/// View space:
+/// -z is forward, +x is right, +y is up
+/// (0.0, 0.0, -1.0) is linear distance of 1.0 in front of the camera's view relative to the camera's rotation
+/// (0.0, 1.0, 0.0) is linear distance of 1.0 above the camera's view relative to the camera's rotation
```

Comment:
Can you define what linear distance is, here?

Reply:
Linear distance refers to the Euclidean distance between two points in a Cartesian coordinate system. I'm not sure if including that would help though. All the sudden I would find myself defining the whole hierarchy of concepts that lead to what a coordinate system is. But I'm totally open to ideas.